source/develop/release-management/features/network/allowExplicitDnsConfiguration.html.md

```diff
@@ -0,0 +1,196 @@
+---
+title: AllowExplicitDnsConfiguration
+category: feature
+authors: mmucha
+wiki_category: Feature
+wiki_title: Features/AllowExplicitDnsConfiguration
+wiki_revision_count: 12
+wiki_last_updated: 2015-10-14
+feature_name: Allow Explicit Dns Configuration
+feature_modules: Networking
+feature_status: Design
+rfe: https://bugzilla.redhat.com/show_bug.cgi?id=1160667
+
+---
+
+# Allow Explicit DNS Configuration
+
+### Owner
+
+*   Name: [ Martin Mucha](User:mmucha)
+*   Email: mmucha@redhat.com
+
+## Summary
+When a new host is added to the system, management network will be
+created for it. As of ovirt-4.0.3, DNS configuration for such network
+will be obtained from resolv.conf file. With this feature implemented,
+an admins should be able to specify overriding configuration in
+WebAdmin. They can do that at several places:
+
+* During creation new host
+* By updating management network of given host (in backend terms — via
+its NetworkAttachment)
+* By updating management network of given DC (or later of given Cluster) 
+
+## Detailed Description
+
+So DNS configuration can be stored in multiple places. If DNS
+configuration is not specified in either of them, no DNS configuration
+data will be passed to VDSM during SetupNetworks command and default
+values from resolv.conf will be used. If either of them is used, it
+will be passed to VDSM overriding configuration from resolv.conf. For
+example we can set DNS configuration in Network related to certain DC
+(or later to certain Cluster) and this configuration will override
+configuration in resolv.conf. If we now setup configuration in
+NetworkAttachment, attaching ManagementNetwork of given DataCenter (or
+later of given Cluster) to specific Host, this configuration is more
+specific than one in Network of DC (or later Cluster) scope and will
+be used instead.  
+
+### DB
+Database needs to be updated so it can accommodate DNS configuration.
+Two places will be altered:
+
+* `network_attachments` table
+* network table
```

Comment:
Mostly English nits, but a quite important change is needed all over the doc: we do not allow setting the DNS during add host flow. All references to this (including the image) should be dropped.